sbt-plugin-test/project/build.sbt

```diff
@@ -1,2 +1,5 @@
-addSbtPlugin("org.scala-js" % "sbt-scalajs" %
-  org.scalajs.ir.ScalaJSVersions.current)
+val scalaJSVersion = org.scalajs.ir.ScalaJSVersions.current
+
+addSbtPlugin("org.scala-js" % "sbt-scalajs" % scalaJSVersion)
+
+libraryDependencies += "org.scala-js" %% "scalajs-linker" % scalaJSVersion
```

Comment:
Why is this necessary? Can we get rid of it? It wouldn't be nice to lose our only test that executes the linker in real real conditions, i.e., in the same conditions as our users.

Reply:
It is for MemOutputFile in the concurrent linker test, so we can't get rid of it. How about I'll onboard us onto scripted and move that particular part to a scripted test (keeping the rest in `sbt-plugin-test`)?

Reply:
Yeah, looks like we really need scripted ...

Although if it's just this particular problem, maybe we can use a file on disk instead of an in memory file? Since we already have a means to create a disk output file with `scalaJSLinkerImpl`?

Reply:
Ah yes. That's fair. OTOH, this neatly tested the class loader magic to make sure it is actually possible to mix two "loadings" of the linker. I think in general this would benefit from some more splitting, so how about we try the scripted approach and otherwise use a real file here?

Reply:
Sounds good